### PR TITLE
Showing communications on activity log

### PR DESCRIPTION
**Pull request by electrified**

### CODE REVIEW

File: src/server/config/types.ts [hunk 1]
```diff
@@ -89,24 +89,30 @@ export interface WellKnownAppointmentTypeMeta {
...
-  PhoneCall = 'phone-call',
-  TextMessage = 'text-message',
+  EmailTextToOffender = 'email-text-to-offender',
+  EmailTextFromOffender = 'email-text-from-offender',
+  EmailTextToOther = 'email-text-to-other',
+  EmailTextFromOther = 'email-text-from-other',
+  PhoneCallToOffender = 'phone-call-to-offender',
+  PhoneCallFromOffender = 'phone-call-from-offender',
+  PhoneCallToOther = 'phone-call-to-other',
+  PhoneCallFromOther = 'phone-call-from-other',
```
Comment (axle-h) [inline src/server/config/types.ts:99]: looks like there's no provision here to support displaying a summarised list on the creation page? Something we can solve later by creating a communication logical group by communication category.. or sometihng.

File: src/server/config/types.ts [hunk 2]
```diff
@@ -89,24 +89,30 @@ export interface WellKnownAppointmentTypeMeta {
...
   name: string
   code: string
 }
 
-export enum WellKnownContactTypeCategory {
+export enum ContactTypeCategory {
   Appointment = 'appointment',
   Communication = 'communication',
+  Other = 'other',
 }
```
Comment (axle-h) [inline src/server/config/types.ts:111]: 👍 , makes sense to me => the fact something is 'well known' doesn't matter to the service & it removes all that null checking around type (which was a mistake...)

**Review by axle-h — APPROVED** [review]

File: src/server/config/types.ts (hunk 1 above)
Comment (electrified) [reply src/server/config/types.ts:99]: We went down this route originally

Comment (electrified) [reply src/server/config/types.ts:99]: We went down this route originally

Yes. We originally had the config like this, to try and support the current designs where you pick the type, sender and recipient but ripped it out:

**Pull request merged**